### PR TITLE
602 convert upload

### PR DESCRIPTION
**Pull request by kindly**

### CODE REVIEW

**Comment by Bjwebb** [comment]
root_list_path should be `iati-activity`, see the example at https://flatten-tool.readthedocs.io/en/latest/usage-iati/

**Comment by Bjwebb** [comment]
This is looking good, a couple of quick observations:
* I think it might be good to have a more explicit license/attribution for the bootswatch theme
* My cursor doesn't change to a pointer when I hover over the accordions.

**Comment by Bjwebb** [comment]
Looking good. I think we should also be including this copyright file in our source tree https://github.com/thomaspark/bootswatch/blob/gh-pages/LICENSE

File: cove_360/sass/_bootstrap-variables-360.scss [hunk 1]
```diff
@@ -1,6 +1,8 @@
 $bootstrap-sass-asset-helper: false !default;
 // Flatly 3.3.6
+// Bootswatch
 // Variables
+// MIT License (see LICENCE.srt)
```
Comment (Bjwebb) [inline cove_360/sass/_bootstrap-variables-360.scss]: Very minor point, put this is a typo `srt` not `rst`

**Comment by Bjwebb** [comment]
I've merged master into this branch so I can use it as the basis for IATI validation work. I've broken some OCDS functionality though. @edugomez if you're looking for something else to do this afternoon, could you have a go at fixing this please.

**Comment by edugomez** [comment]
@Bjwebb  this now fails due to coveralls, tests are fixed. Is this good to go?

**Pull request merged**